test/temporal.jl

```diff
@@ -163,14 +164,20 @@
 
             @testset "dims = 1" begin
                 d = 1
-                expected = [[9, 10], [10, 11]]
-                @test Transforms.apply(A, hod; dims=d, inds=[2, 3]) == expected
+                expected = [2 10 11]
+                @test Transforms.apply(A, hod; dims=d, inds=[2]) == expected
+
+                expected = [1 9 10; 2 10 11]
+                @test Transforms.apply(M, hod; dims=d, inds=[1, 2]) == expected
             end
 
             @testset "dims = 2" begin
                 d = 2
-                expected = [[2], [10], [11]]
-                @test Transforms.apply(A, hod; dims=d, inds=[2]) == expected
+                expected = [9 10; 10 11]
```

Comment:
this is very different to the answer above - in spite of the different `inds`. What caused this?

Reply:
Not sure what you mean by answer above, but I'll explain how I checked this.

```
1  9   10
2  10  11
```

`dims=2` slices into rows `[1, 9, 10]` and `[2, 10, 11]`. `inds = [2, 3]` picks the 2nd and 3rd element of each of those rows which is `9, 10` and `10, 11` respectively.

I thought this same logic checked out for the other test cases but let me know if you think otherwise, or it's not how you expect it to work.

Reply:
>Not sure what you mean by "answer above",

Sorry, I meant in relation to the expected answer it's replacing: `expected = [[2], [10], [11]]`

Before this commit, for `d=2`, `inds=[2]` the result was `[[2], [10], [11]]` a `Vector{Vector}`

Now, for `d=2`, `inds=[2, 3]` the result is `[9 10; 10 11]` an `Array{Int, 2}`.

It's correct, but I'm wondering why it was so different before. I'm guessing it's the result of now using `mapslices`?

Reply:
Ah, I was confused because I was looking at the full diff.

Reply:
The dims were in the opposite order before thanks to eachslice versus mapslices differences